sysboundary/sysboundary.cpp

```diff
@@ -268,15 +268,15 @@ bool SysBoundary::initSysBoundaries(Project& project, creal& t) {
          if((faces[4] || faces[5]) && isPeriodic[2]) {
             if(myRank == MASTER_RANK) cerr << "You set boundaries.periodic_z = yes and load Static system boundary conditions on the z+ or z- face, are you sure this is correct?" << endl;
          }
-         if((faces[0] || faces[1]) && P::xcells_ini < 5) {
```

Comment:
Perhaps here instead of changing the 5 to a one, we keep it at 5 but also set a check if maxreflevel is 0 or not.